Core/scripts/dirac-install.py

```diff
@@ -1352,6 +1352,9 @@ def createBashrc():
                      '( echo $PYTHONPATH | grep -q $DIRAC ) || export PYTHONPATH=$DIRAC:$PYTHONPATH'] )
       lines.extend( ['# new OpenSSL version require OPENSSL_CONF to point to some accessible location',
                      'export OPENSSL_CONF=/tmp'] )
+      lines.extend( ['# CAs path for SSL verification',
```

Comment:
Correct. Sorry, I didn't see you message before merging, I will make a further change. Thanks